### ISSUE TITLE
celery start: ImportError: No module named mist.io

### ISSUE THREAD

**Issue by mrmikee** [mistio/mist.io#751]
Fresh Ubuntu 16.04 server install following Installation instructions in Readme.rst file.
After applying the [buildout patch from X4 (re: issue740)](https://github.com/mistio/mist.io/issues/740#issuecomment-217352723):

```
You can easily fix the whole buildout drama, by replacing line 38-58 in buildout.cfg with the config below:

[mist]
recipe = zc.recipe.egg
dependent-scripts = true
eggs =
    mist.io
    PasteScript
    ${buildout:eggs}

[uwsgi]
recipe = unweb.recipe.uwsgi
url = http://projects.unbit.it/downloads/uwsgi-2.0.11.2.tar.gz
eggs =
    ${mist:eggs}

[pip]
recipe = collective.recipe.cmd
shell = /bin/bash
on_install = true
on_update = true
cmds = ./bin/pip install -r requirements.txt; echo $'\npip modules installed.\n'
```

The install goes well, but once I run `./bin/supervisord`.
Celery and uwsgi fails to start.

``` bash
./bin/supervisorctl status
celery                           FATAL     Exited too quickly (process log may have details)
haproxy                          RUNNING   pid 30013, uptime 0:05:44
hub-shell                        STARTING  
memcache                         RUNNING   pid 30019, uptime 0:05:44
rabbitmq                         RUNNING   pid 30016, uptime 0:05:44
sockjs                           STARTING  
uwsgi                            FATAL     Exited too quickly (process log may have details)
```

Inspection of the log file shows:

```
cat var/log/celery-stderr---supervisor-MOew4j.log 
Traceback (most recent call last):
  File "/home/mc4081/www/mist.io/bin/celery", line 11, in <module>
    sys.exit(main())
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/__main__.py", line 30, in main
    main()
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/celery.py", line 81, in main
    cmd.execute_from_commandline(argv)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/celery.py", line 769, in execute_from_commandline
    super(CeleryCommand, self).execute_from_commandline(argv)))
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 305, in execute_from_commandline
    argv = self.setup_app_from_commandline(argv)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 465, in setup_app_from_commandline
    self.app = self.find_app(app)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 485, in find_app
    return find_app(app, symbol_by_name=self.symbol_by_name)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/app/utils.py", line 229, in find_app
    sym = symbol_by_name(app, imp=imp)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 488, in symbol_by_name
    return symbol_by_name(name, imp=imp)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/kombu/utils/__init__.py", line 96, in symbol_by_name
    module = imp(module_name, package=package, **kwargs)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/utils/imports.py", line 101, in import_from_cwd
    return imp(module, package=package)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named mist.io
Traceback (most recent call last):
  File "/home/mc4081/www/mist.io/bin/celery", line 11, in <module>
    sys.exit(main())
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/__main__.py", line 30, in main
    main()
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/celery.py", line 81, in main
    cmd.execute_from_commandline(argv)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/celery.py", line 769, in execute_from_commandline
    super(CeleryCommand, self).execute_from_commandline(argv)))
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 305, in execute_from_commandline
    argv = self.setup_app_from_commandline(argv)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 465, in setup_app_from_commandline
    self.app = self.find_app(app)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 485, in find_app
    return find_app(app, symbol_by_name=self.symbol_by_name)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/app/utils.py", line 229, in find_app
    sym = symbol_by_name(app, imp=imp)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 488, in symbol_by_name
    return symbol_by_name(name, imp=imp)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/kombu/utils/__init__.py", line 96, in symbol_by_name
    module = imp(module_name, package=package, **kwargs)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/utils/imports.py", line 101, in import_from_cwd
    return imp(module, package=package)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named mist.io
Traceback (most recent call last):
  File "/home/mc4081/www/mist.io/bin/celery", line 11, in <module>
    sys.exit(main())
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/__main__.py", line 30, in main
    main()
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/celery.py", line 81, in main
    cmd.execute_from_commandline(argv)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/celery.py", line 769, in execute_from_commandline
    super(CeleryCommand, self).execute_from_commandline(argv)))
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 305, in execute_from_commandline
    argv = self.setup_app_from_commandline(argv)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 465, in setup_app_from_commandline
    self.app = self.find_app(app)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 485, in find_app
    return find_app(app, symbol_by_name=self.symbol_by_name)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/app/utils.py", line 229, in find_app
    sym = symbol_by_name(app, imp=imp)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 488, in symbol_by_name
    return symbol_by_name(name, imp=imp)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/kombu/utils/__init__.py", line 96, in symbol_by_name
    module = imp(module_name, package=package, **kwargs)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/utils/imports.py", line 101, in import_from_cwd
    return imp(module, package=package)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named mist.io
Traceback (most recent call last):
  File "/home/mc4081/www/mist.io/bin/celery", line 11, in <module>
    sys.exit(main())
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/__main__.py", line 30, in main
    main()
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/celery.py", line 81, in main
    cmd.execute_from_commandline(argv)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/celery.py", line 769, in execute_from_commandline
    super(CeleryCommand, self).execute_from_commandline(argv)))
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 305, in execute_from_commandline
    argv = self.setup_app_from_commandline(argv)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 465, in setup_app_from_commandline
    self.app = self.find_app(app)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 485, in find_app
    return find_app(app, symbol_by_name=self.symbol_by_name)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/app/utils.py", line 229, in find_app
    sym = symbol_by_name(app, imp=imp)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 488, in symbol_by_name
    return symbol_by_name(name, imp=imp)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/kombu/utils/__init__.py", line 96, in symbol_by_name
    module = imp(module_name, package=package, **kwargs)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/utils/imports.py", line 101, in import_from_cwd
    return imp(module, package=package)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named mist.io
Traceback (most recent call last):
  File "/home/mc4081/www/mist.io/bin/celery", line 11, in <module>
    sys.exit(main())
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/__main__.py", line 30, in main
    main()
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/celery.py", line 81, in main
    cmd.execute_from_commandline(argv)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/celery.py", line 769, in execute_from_commandline
    super(CeleryCommand, self).execute_from_commandline(argv)))
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 305, in execute_from_commandline
    argv = self.setup_app_from_commandline(argv)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 465, in setup_app_from_commandline
    self.app = self.find_app(app)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 485, in find_app
    return find_app(app, symbol_by_name=self.symbol_by_name)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/app/utils.py", line 229, in find_app
    sym = symbol_by_name(app, imp=imp)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/bin/base.py", line 488, in symbol_by_name
    return symbol_by_name(name, imp=imp)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/kombu/utils/__init__.py", line 96, in symbol_by_name
    module = imp(module_name, package=package, **kwargs)
  File "/home/mc4081/www/mist.io/local/lib/python2.7/site-packages/celery/utils/imports.py", line 101, in import_from_cwd
    return imp(module, package=package)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named mist.io
```

**Comment by mgogoulos** [comment]
Didn't have the chance to try the patch proposed. On the meanwhile, if you cannot install mist.io on a previous Ubuntu version (eg 14.04) you can try out the docker image - http://docs.mist.io/article/39-installation

**Comment by mrmikee** [comment]
Docker image times out behind a proxy due to git failure to connect in `/init.sh`, but eventually runs after a VERY long wait.

Also, don't care to run the overhead/headaches of docker in production. Hope the buildout method gets fixed for 16.04 installs. Thanks for your time.

**Comment by mgogoulos** [comment]
Version 2 is out, comes with an easier installation process using Docker Compose and many new features, check updated instructions on https://github.com/mistio/mist.io
